python/py-kitchen/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-kitchen
+version             1.2.6
+supported_archs     noarch
+license             LGPL-21
+
+python.versions     37 38 39
```

Comment:
Current Python version for MacPorts is 3.10, so you should also have `310` here as well.  Were you intentionally excluding 3.10?

Reply:
Hi @herbygillot,

I omitted Python 3.10 accidentally.  I've added it to the Portfile.

Thanks,

-ranga